web_widget_digitized_signature/__manifest__.py

```diff
@@ -1,18 +1,20 @@
 # Copyright 2004-2010 OpenERP SA (<http://www.openerp.com>)
 # Copyright 2011-2015 Serpent Consulting Services Pvt. Ltd.
 # Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright (C) 2019 - TODAY, Open Source Integrators
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
-    "name": "Web Widget Digitized Signature",
-    "version": "11.0.1.0.0",
-    "author": "Serpent Consulting Services Pvt. Ltd., "
-              "Agile Business Group, "
-              "Tecnativa, "
-              "Odoo Community Association (OCA)",
+    'name': 'Web Widget Digitized Signature',
+    'version': '12.0.1.0.0',
+    'author': 'Serpent Consulting Services Pvt. Ltd., '
+              'Agile Business Group, '
+              'Tecnativa, '
+              'Open Source Integrators,'
```

Comment:
You are not doing a significant contribution to the module, so you mustn't put in authors. Only on contributors.